lib/compiler/src/function.rs

```diff
@@ -35,7 +36,7 @@ pub struct CompiledFunctionFrameInfo {
 
 /// The function body.
 #[cfg_attr(feature = "enable-serde", derive(Deserialize, Serialize))]
-#[derive(Debug, Clone, PartialEq, Eq)]
+#[derive(Debug, Clone, PartialEq, Eq, MemoryUsage)]
```

Comment:
serde is disabled here because it requires `std` I think. If `MemoryUsage` does too, we probably want it to be grouped with the `serde` things, not with the rest. Though the naming of `enable-serde` makes me more uncertain what the intent is here.

Reply:
`enable-serde` isn't related to `std` or `core`/`alloc`. It's just we allowed to run without Serde in the past. Not sure it's still useful today though 🤔.